completions/mdadm

```diff
@@ -106,40 +105,37 @@ _comp_cmd_mdadm()
 
     if [[ $cur == -* ]]; then
         if ((cword == 1)); then
-            COMPREPLY=($(compgen -W "$options --assemble --build --create
-                --monitor --grow" -- "$cur"))
+            _comp_compgen -- -W "$options --assemble --build --create --monitor
+                --grow"
```

Comment:
Maybe, per https://github.com/scop/bash-completion/pull/961#discussion_r1186844166
```suggestion
            _comp_compgen -- -W '$options --assemble --build --create --monitor
                --grow'
```

...and similarly for all modifications in this file below?